src/main/java/net/minecraftforge/common/data/ForgeItemTagsProvider.java

```diff
@@ -54,7 +53,6 @@ public void registerTags()
         super.registerTags();
         filter = this.tagToBuilder.entrySet().stream().map(e -> e.getKey()).collect(Collectors.toSet());
 
-        func_240522_a_(Tags.Items.ARROWS).func_240534_a_(Items.ARROW, Items.TIPPED_ARROW, Items.SPECTRAL_ARROW);
         func_240522_a_(Tags.Items.BONES).func_240534_a_(Items.BONE);
```

Comment:
Same here, it shouldn't just dissapear, if we want to remove it, mark it for removal in 1.16.2.